addons/partner_autocomplete/static/src/js/web_company_autocomplete.js

```diff
@@ -0,0 +1,27 @@
+odoo.define('web.company_autocomplete', function (require) {
+"use strict";
+
+const AbstractWebClient = require('web.AbstractWebClient');
+const session = require('web.session');
+
+return AbstractWebClient.include({
+
+    start: function () {
+        console.log(session);
+        if (session.iap_company_enrich) {
+            const current_company_id = session.user_companies.current_company[0];
+            this._rpc({
+                model: 'res.company',
+                method: 'iap_enrich_auto',
+                args: [current_company_id],
+            }, {
+                shadow: true,
+            });
+        }
```

Comment:
So any time anyone logs in the backend it triggers an enrichment check on their main company through an RPC?

Reply:
Nope, only one call is performed then deactivated.